changelog/unreleased/issue-3490

```diff
@@ -0,0 +1,10 @@
+Enhancement: Support for specifying file size in read-data-subset
+
+To check a subset of repository files, restic used to support two ways to
+select a subset - A specific range of pack files, or random percentage of 
+pack files. I have added a third method to select pack files - By specifying 
+file size. This new option is available with the 'restic check' command.

```

Comment:
The `check` command should be mentioned in the first sentence of the changelog. Otherwise it will be hard for a user to understand the explanations.